src/views/tickets/view-link.php

```diff
@@ -29,21 +30,25 @@
 
 $is_event_page = class_exists( 'Tribe__Events__Main' ) && Tribe__Events__Main::POSTTYPE === $event->post_type;
 
-$events_label_singular = $post_type->labels->singular_name;
-$counters              = [];
-$rsvp_count   = $view->count_rsvp_attendees( $event_id, $user_id );
-$ticket_count = $view->count_ticket_attendees( $event_id, $user_id );
+$post_type_singular = $post_type ? $post_type->labels->singular_name : _x( 'Post', 'fallback post type singular name', 'event-tickets' );
+$counters           = [];
+$rsvp_count         = $view->count_rsvp_attendees( $event_id, $user_id );
+$ticket_count       = $view->count_ticket_attendees( $event_id, $user_id );
 
 if ( 1 === $rsvp_count ) {
-	$counters[] = sprintf( _x( '%1d %2s', 'RSVP count singular', 'event-tickets' ), $rsvp_count, tribe_get_rsvp_label_singular( basename( __FILE__ ) ) );
+	// Translators: 1: the number one, 2: singular RSVP label.
```

Comment:
Also for these, I'd use `%1d` and `%2s` in the comments. Translators are normally non tech people and maybe they need some literal stuff to interpret it.

Reply:
See https://developer.wordpress.org/apis/handbook/internationalization/internationalization-guidelines/#variables